modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -1,47 +1,105 @@
+.. In *Enter measurements* select *Edit inline* to enter the name and value for the metric.
+.. In the *Fact* dialog box, enter a *Name* and numerical *Value* for the fact. +
+[NOTE]
+For facts, the value must always be numerical. This is not validated at design time, but it will cause a runtime exception once deployed.
+.. Leave the *Collapsible* option with the default of *True*.
+.. Click *Finish*. +
+You can repeat these steps to add multiple key values. +
+This image shows example values for a custom metric that tracks t-shirt sales.
++
+image::custom-metrics-connector-config.png[]
++
+. Select the *Throw runtime exception on validation failure* option if you want the flow to break if invalid values are entered for the metric. If this option is left unchecked, an error is will be logged, but the flow won't break.
+
+== Deploy the Application
+
+. Deploy the application to a supported standalone Mule runtime engine.
+. In Anypoint Monitoring, xref:dashboard-custom-config.adoc[add a new custom dashboard] for the custom metric. +
+This image shows an example of the configuration for a graph to display t-shirt sales.
++
+image::custom-metric-custom-dashboard-config.png[]
++
 [NOTE]
-You must select a hybrid application (application deployed to an on-premises server and managed in the cloud) to see the *Custom metrics* tab. +
-If the Custom Metrics connector has collected one or more metrics in the specified time range, one chart per defined metric is displayed. 
-+
-image::custom-metrics-dashboard.png[]
-+
-. Click *Select a measurement for this chart* to display the measurement for the chart.
-. Use the drop-down lists at the bottom of the chart to filter the data by: +
-* *Aggregation* - Possible aggregations include:
-** *None* (default) - No data aggregation.  
-** *count* - Displays the total number of items collected for the measurement.
-** *distinct* - Returns the unique values for the given measurement.
-** *integral* - Returns the area under the curve for subsequent values.
-** *mean* - Returns the arithmetic average of the specified measurement over a given interval of time.
-** *median* - Displays the middle value for the collected metrics.
-** *mode* - Displays the most frequent value in the metrics collected for the defined measurement. 
-** *sum* - Displays the sum of all the values for the defined measurement. 
-* *Measurement* - The choices in this list are determined by the measurement values you define in the Custom Metrics connector. 
-* *Group by* - The choices in this list are determined by the values you define in the Custom Metrics connector. 
-
-
-=== See Also
-
-* xref:connectors::metrics/custom-metrics-connector.adoc[Custom Metrics Connector] 
-* xref:connectors::metrics/custom-metrics-connector-reference.adoc[Custom Metrics Connector Reference]
\ No newline at end of file
+You only see the custom metrics in the pre-populated metric selection drop-down lists once something is logged for the custom metric. 
```

Comment:
really long noun phrases like "the pre-populated metric selection drop-down lists" are hard to read. Can you break it up, like "the drop-down lists for metric selection" (of course they are pre-populated, that's what a drop-down list is?)...

Reply:
That's the thing--if nothing has been logged for the custom metric, the drop-down list appears, but there will be nothing there (not populated). So the list is based on the metrics logged.